src/components/Telephone/contacts.js

```diff
@@ -65,20 +78,9 @@ export const contactsMap = Object.freeze({
     phoneNumber: '8888263127',
     description: 'U.S. Department of the Treasury (Debt Management Services)',
   },
-  FEDERAL_RELAY_SERVICE: {
-    phoneNumber: '8008778339',
-    description: 'Federal Relay Service',
-  },
-  SUICIDE_PREVENTION_LIFELINE: {
-    phoneNumber: '8007994889',
-    description: 'Suicide Prevention Line',
-  },
-  DMDC_DEERS: {
-    phoneNumber: '8663632883',
-    description:
-      'Defense Manpower Data Center (DMDC) | Defense Enrollment Eligibility Reporting System (DEERS) Support Office',
-  },
-  VA_311: { phoneNumber: '8006982411', description: 'VA Help desk (VA311)' },
+  // VA_311 used before the number changed to include 411
+  VA_311: { phoneNumber: '8006982411', description: 'VA Help desk (VA411)' },
+  VA_411: { phoneNumber: '8006982411', description: 'VA Help desk (VA411)' },
```

Comment:
I figured `VA_411` makes more sense... we can now remove the older `VA311` at our leasure.